src/scrapers/platformen/scraper.py

```diff
@@ -226,3 +234,28 @@ def add_initiative(self, initiative):
 
     def get_logger(self) -> logging.Logger:
         raise NotImplementedError("Should be implemented by derived scraper")
+
+    def set_group(self, group):
+        """
+        Restricts the scraper to a certain group. If it's not supported it
+        will only log an error message and be ignored this scraping will proceed!
+        """
+        supports_group = self.supports_group(group)
+        if not supports_group:
+            if supports_group is None:
+                self.get_logger().warning(f"{self.name} does not support restricting on groups!")
+            else:
+                self.get_logger().error(f"{self.name} does not support {group}!")
+        else:
+            self._group = group
+
+    def supports_group(self, group):
+        """
+        Implement this to indicate the scraper has support to restrict the scraping
+        to one group. Return none to indicate when it has the possibility to filter
```

Comment:
Vanuit de set_group logica hierboven begreep ik niet zo goed dat er eigenlijk 3 states zijn. Misschien zou je hier een Enum voor kunnen gebruiken, ipv True/False/None

Reply:
Ik heb het met een exception opgelost. Ik ben met je eens dat het niet helemaal helder was. Eigenlijk gaat het er gewoon om dat het wel of niet geïmplementeerd is.